pkgs/development/compilers/gcc/common/builder.nix

```diff
@@ -7,7 +7,7 @@
 
 let
   forceLibgccToBuildCrtStuff = import ./libgcc-buildstuff.nix { inherit lib stdenv; };
-  isCross = with stdenv; targetPlatform.config != hostPlatform.config;
```

Comment:
These really should be `lib.systems.equals`, all of them, now that I think about it.

Reply:
Just those three or everything in gcc? A quick `git grep 'Platform *[!=]=' | wc -l` inside `pkgs/development/compilers/gcc` gives me 40 hits.

Reply:
Generally, all of them in nixpkgs, technically...

Reply:
Sure, but I don't want to let this single line change scope creep into doing a treewide update.
I've replaced things inside gcc, a simple rebuild of `nix-build -A gcc` was a no-op, but I'm not sure how to test this properly.